packages/manager/modules/pci/src/projects/project/kubernetes/add/add.html

```diff
@@ -90,6 +101,26 @@ <h1 data-translate="kubernetes_add"></h1>
         </node-pool-configuration>
     </oui-step-form>
 
+    <!--Autoscaling-->
```

Comment:
How is autoscaling related to adding non-available regions? Is this change supposed to be a part of this commit?

Reply:
Same as my previous comment.
It's here because i got a conflicts when i rebase on fresh develop branch.

Reply:
Completed and thanks